### PR TITLE
✨ Connect to MongoDB

### PR DESCRIPTION
**Pull request by oersoy91**
- ➕ Install mongodb and dotenv

- 🙈 Ignore dotenv

- ✨ Insert Item into collection

- ✨  Connect Next.js API to MongoDB

### CODE REVIEW

File: components/button/Button.module.css [hunk 1]
```diff
@@ -1,8 +1,4 @@
 .btn {
-  border-radius: 10px;
-}
-
-.primary {
-  color: antiquewhite;
-  background: black;
+  width: 200px;
+  height: 100px;
```
Comment (phlgr) [inline components/button/Button.module.css:3]: I would not declare absolute `width` and `height` as the button width could be controlled by the content

File: components/logo/Logo.module.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.logo {
+  width: 200px;
+  height: 300px;
```
Comment (phlgr) [inline components/logo/Logo.module.css:3]: Same as with the button

**Comment by phlgr** [comment]
GJ 🚀

File: server/db.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+export async function getCollection(collectioName) {
+  return await db.collection(collectioName);
+}
+
+export function closeDB() {
+  client.close();
+}
+
+export async function createNewPersonnel(personnel) {
+  const personnelCollection = await getCollection("test1111");
```
Comment (miriamhaenle) [inline server/db.ts:41]: Why not call your collection 'personnel' ?

**Review by miriamhaenle — COMMENTED** [review]
lgtm 👍

**Pull request merged**